### DISCUSSION TITLE
subtle flashing of whole screen when mouse stops moving on the iced app window

### DISCUSSION THREAD

**Discussion by correabuscar** [hecrj/iced#1770] [General]
Heya, whilst trying to do some learning in project https://github.com/correabuscar/iced_multitest2 I've noticed a very very subtle flashing going on only when mouse stops moving on top the window contents of that iced app and it seems to happen on any iced app.

 This flashing happens on the whole screen on Windows 11 Pro 22H2, possibly due to some settings i've in windows, i don't know, but it doesn't happen when moving mouse outside of the iced window, or on other software.

 The flashing seems to be a very slight increase in brightness and then comes back to normal. It happens very fast though. It's almost like you don't see it at all, if you're not paying attention.

 When you start moving mouse and you keep moving it, no flashing seems to happen. But as soon as you stop moving the mouse, you notice one or two fast flashes like that. So moving mouse left to right repeatedly should generate some flashes.

 Not sure how to explain it better, but can anyone try that project(it's simple/harmless enough) to see if they see the flashing? It doesn't happen on Linux.

**Comment by correabuscar** [comment]
I've finally tracked this down and i've explained it here(somewhat): https://github.com/correabuscar/glutin_simple1
**tl;dr:** it's not an iced issue, it's a g-sync + freesync-extended + swap_buffers()/glutin thing. (all 3 at once, required to reproduce it)
